src/sentry/eventtypes/error.py

```diff
@@ -44,11 +44,19 @@ def extract_metadata(self, data):
         return rv
 
     def compute_title(self, metadata):
+        # Both of those properties are set within EventManager
+        if "finest_tree_label" in metadata:
+            return " | ".join(metadata["finest_tree_label"])
+
+        if "current_tree_label" in metadata:
```

Comment:
Is it even possible that `current_tree_label` is present while `finest_tree_label` is not? If not, should we remove this branch to make it more clear that `current_tree_label` is for the group and `finest_tree_label` is for the event?

Reply:
Group title computation and event title computation still go through the same function so we need to handle both cases here.

I forgot to remove the models code, fixing